akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/RouteDirectivesSpec.scala

```diff
@@ -133,16 +136,16 @@ class RouteDirectivesSpec extends FreeSpec with GenericRoutingSpec {
 
   case class Data(name: String, age: Int)
   object Data {
-    //import spray.json.DefaultJsonProtocol._
-    //import spray.httpx.SprayJsonSupport._
+    import spray.json.DefaultJsonProtocol._
+    import SprayJsonSupport._
 
-    val jsonMarshaller: ToEntityMarshaller[Data] = FIXME // jsonFormat2(Data.apply)
-    val xmlMarshaller: ToEntityMarshaller[Data] = FIXME
-    /*Marshaller.delegate[Data, xml.NodeSeq](MediaTypes.`text/xml`) { (data: Data) ⇒
-        <data><name>{ data.name }</name><age>{ data.age }</age></data>
-      }*/
+    val jsonMarshaller: ToEntityMarshaller[Data] = jsonFormat2(Data.apply)
+    val xmlMarshaller: ToEntityMarshaller[Data] = Marshaller.withOpenCharset(MediaTypes.`text/xml`) { (data: Data, charset: HttpCharset) ⇒
+      val ct = ContentType(MediaTypes.`text/xml`)
+      HttpEntity(ct, <data><name>{ data.name }</name><age>{ data.age }</age></data>.toString())
```

Comment:
This is not nice; Still wrapping my head around Marshallers here, looking at spray now how delegate was implemented in order to use `ScalaXmlSupport`'s provided marshallers here somehow